selinux/authlogin_duo.te

```diff
@@ -0,0 +1,31 @@
+module authlogin_duo 2.0.0;
+
+## <desc>
+##  <p>
+##  Allow sshd to use the pam_duo PAM module.
+##  </p>
+## </desc>
+gen_tunable(pam_duo_permit_sshd, false)
```

Comment:
can we turn this to true by default?
that will help us maintain backwards compatibility